fixedwidth/src/main/java/org/apache/metamodel/fixedwidth/FixedWidthConfiguration.java

```diff
@@ -33,66 +33,74 @@
 /**
  * Configuration of metadata about a fixed width values datacontext.
  */
-public final class FixedWidthConfiguration extends BaseObject implements
-		Serializable {
+public final class FixedWidthConfiguration extends BaseObject implements Serializable {
 
-	private static final long serialVersionUID = 1L;
+    private static final long serialVersionUID = 1L;
 
-	public static final int NO_COLUMN_NAME_LINE = 0;
-	public static final int DEFAULT_COLUMN_NAME_LINE = 1;
+    public static final int NO_COLUMN_NAME_LINE = 0;
+    public static final int DEFAULT_COLUMN_NAME_LINE = 1;
 
-	private final String encoding;
-	private final int fixedValueWidth;
-	private final int[] valueWidths;
-	private final int columnNameLineNumber;
-	private final boolean failOnInconsistentLineWidth;
-	private final ColumnNamingStrategy columnNamingStrategy;
+    private final String encoding;
+    private final int fixedValueWidth;
+    private final int[] valueWidths;
+    private final int columnNameLineNumber;
+    private final boolean failOnInconsistentLineWidth;
+    private final boolean headerPresent;

```

Comment:
@LosD: I do it generally by skipping whitespace until I reach data but in the case of "the one" file we had it was just the length of one record. 

@kaspersorensen: Yes, I could set columnNameLineNumber to 2 but I am not sure if "a common user" would know to do this too. Unless (s)he knows the details of the file which I assume we can not expect. On the other hand if there is "column-names-header" then (s)he has to be aware of it anyway to use it properly. I don't have a strong opinion here and will go with "whatever" you advice. :-)